cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/ComputeEngineCredentials.java

```diff
@@ -123,9 +130,40 @@ private AccessToken refreshAccessTokenRemotely(String endPoint) throws IOExcepti
     throw new IOException(exception.getMessage(), exception);
   }
 
+  private void disableVerifySSL(HttpsURLConnection connection) throws IOException {
+    try {
+      SSLContext sslContext = SSLContext.getInstance("SSL");
```

Comment:
nit: May want to consider using SSLContext.getDefault() to prevent older insecure SSL protocols from being used (if you want to be extra cautious, you should use TLSv1.2+).

Reply:
I wanted to be consistent with how [HttpsEnabler instantiates SSLContext](https://github.com/cdapio/cdap/blob/develop/cdap-common/src/main/java/io/cdap/cdap/common/security/HttpsEnabler.java#L254).